src/slic3r/GUI/Tab.cpp

```diff
@@ -1688,6 +1688,10 @@ void TabPrint::build()
         optgroup->append_single_option_line("arc_fitting");
         optgroup->append_single_option_line("xy_size_compensation");
         optgroup->append_single_option_line("elefant_foot_compensation", "elephant-foot-compensation_114487");
+        optgroup->append_separator();
+        optgroup->append_single_option_line("make_overhang_printable", category_path);
+        optgroup->append_single_option_line("make_overhang_printable_angle", category_path);
+        optgroup->append_single_option_line("make_overhang_printable_hole_size", category_path);
```

Comment:
I don't think you want a "category_path" here, but if you do you should set it.

Reply:
Fixed